modules/shared/src/main/scala/org/tessellation/currency/dataApplication/package.scala

```diff
@@ -308,72 +311,76 @@ object BaseDataApplicationService {
 
       def routesPrefix: ExternalUrlPrefix = v.routesPrefix
     }
-}
 
-object BaseDataApplicationL0Service {
-  def apply[F[_], D <: DataUpdate, DON <: DataOnChainState, DOF <: DataCalculatedState](
-    service: DataApplicationL0Service[F, D, DON, DOF]
-  )(implicit d: ClassTag[D], don: ClassTag[DON], dof: ClassTag[DOF], monadThrow: MonadThrow[F]): BaseDataApplicationL0Service[F] = {
-    val base = BaseDataApplicationService.apply[F, D, DON, DOF, L0NodeContext[F]](service, service)
+  object BaseDataApplicationL0Service {
```

Comment:
if it's already wrapped into a parent service maybe it's enough to name objects `l0` and `l1`?